types/matter-js/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for Matter.js - 0.17.1
+// Type definitions for Matter.js - 0.18.0
```

Comment:
Change according to [this](https://github.com/photonstorm/phaser/commit/1e1ff3651cefe2d1184a024d0729f3f01fc30b98#diff-ef9c2ebb0b1cde4244acd85c62242c2b7094bdab80a8a6f1bc70ad1a57b3128fR5018), which is the latest version as of now.

Reply:
This should actually not include the patch version, because that's what the Definitely Typed publisher uses to distinguish multiple changes to the same major.minor.


```suggestion
// Type definitions for Matter.js - 0.18
```

Reply:
tslint.json has `"dt-header": false,`; if you remove that, then the CI tests will check the format for you.

In general, reducing the number of lint exceptions is a good idea.